docs/CURL-DISABLE.md

```diff
@@ -140,6 +140,10 @@ Disable the built-in progress meter
 
 Disable support for proxies
 
+## `CURL_DISABLE_IPFS`
+
+Disable the IPFS/IPNS protocols.
```

Comment:
I think it is worth mentioning that this affects the curl tool only, as libcurl does not know anything about IPFS.